doc/040_backup.rst

```diff
@@ -45,6 +45,17 @@ size of the files and directories in ``~/work`` on the local file system. It
 also tells us that only 1.200 GiB was added to the repository. This means that
 some of the data was duplicate and restic was able to efficiently reduce it.
 
+We just attached the absolute path ``~/work`` to the backup, so the path
+within the repository is ``/home/user/work``, depending on your user name.
+
+If we attach a relative path ``work``, the path within the repository is ``/work``.
+
+For example ``restic backup work`` run from ``/home/user`` crates a snapshot
+with a attached path ``/home/user/work`` that contains the path ``/work``
```

Comment:
What about `[...] creates a snapshot with the directory /home/user/work stored in the snapshot metadata` plus showing the output of `snapshots`?

Reply:
As the column in the snapshots command is named  Directory that would be consistent.

Should  "attached path" be used at all?